tests/unit/store/settings.spec.js

```diff
@@ -394,15 +397,20 @@ describe("store/settings", () => {
     store.commit("settings/set_customer_account_ids", current_customerids);
 
     /*  assert the uuid and nickname from settings store is updated */
-    let updated_focus_customerid = store.getters["settings/customer_details"];
-    expect(updated_focus_customerid.uuid).toEqual("7N42dgm5tFLK9N8MT7fHC7");
-    expect(updated_focus_customerid.nickname).toEqual("Updated account -2");
+    const updated_focus_customerid =
+      store.state.settings.customer_account_ids[1];
+    expect(updated_focus_customerid.uuid).toStrictEqual(
+      "7N42dgm5tFLK9N8MT7fHC7"
+    );
+    expect(updated_focus_customerid.nickname).toStrictEqual(
+      "Updated account -2"
+    );
   });
-  test("Store a multiple customer detail with multiple user ids can be assigned to the customer_ids array an 'Edit Customer' ==>  PopopUpDialog (DeleteID) ==> SettingsForm (SaveChanges) ", () => {
+  test("Given the store has multiple customer details, When the mutation deletes one Customer details, Then validate the number of the customer decrements by one", () => {
```

Comment:
The mutation is not doing any deleting right?  In the code of this test itself you are deleting an element from the array (this test calls the `splice` function on the array), then the mutation is just taking that array you pass as an argument and committing it to the store.  It may be useful in the future to create a mutation that actually does delete a particular user---but that's not what the current code does, so we don't want to confuse other devs reading the code into thinking that feature is already implemented

Reply:
Would like to retain the present title, its more informative.

Reply:
This is not about a title being informative or not, the current title is inaccurate.  The mutation you are calling does not delete anything.  The mutation you are calling just assigns the array it receives as a function argument to the store.  This test is not testing whether a mutation deletes anything, because the test itself splices the array before passing it to the mutation. It is critical that the test title describe what is happening so that when people look at the code later to understand how it works they aren't mislead about what a test is doing or what activities a mutation performs.